src/coveralls.erl

```diff
@@ -110,9 +110,11 @@ send(Json, #s{poster=Poster, poster_init=Init}) ->
   Type     = "multipart/form-data; boundary=" ++ Boundary,
   Body     = to_body(Json, Boundary),
   R        = Poster(post, {?COVERALLS_URL, [], Type, Body}, [], []),
+  io:format("coveralls returns: ~p~n", [R]),
```

Comment:
Nit: was this left intentionally?

Reply:
Yes. It's good to see what exactly we've got from coveralls.